packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1343,3 +1352,157 @@ export function listenToEventResponderEventTypes(
 if (enableEventAPI) {
   setListenToResponderEventTypes(listenToEventResponderEventTypes);
 }
+
+const emptyObject = {};
+
+export function createEventTargetHitSlop(
+  left: number | void,
+  right: number | void,
+  top: number | void,
+  bottom: number | void,
+  rootContainerInstance: Element | Document,
+  parentNamespace: string,
+): Element {
+  const hitSlopElement = createElement(
+    'div',
+    emptyObject,
+    rootContainerInstance,
+    parentNamespace,
+  );
+  const hitSlopElementStyle = ((hitSlopElement: any): HTMLElement).style;
+
+  hitSlopElementStyle.position = 'absolute';
+  hitSlopElementStyle.display = 'block';
+  if (top !== undefined) {
```

Comment:
The value could also be `null`. When a `value == null` we should default to `0px`